src/algorithms/SortingAlgorithms.java

```diff
@@ -19,11 +21,30 @@ public static void sortByBubbleSorting(int[] array) {
     }
 
     public static void sortBySelectionSorting(int[] array){
```

Comment:
It doesn't work correctly
https://en.wikipedia.org/wiki/Selection_sort

Reply:
Thanks for valuable comment!